port/ztpf/omrintrospect.c

```diff
@@ -1079,12 +1079,12 @@ omrintrospect_threads_startDo_with_signal(struct OMRPortLibrary *portLibrary, J9
  *	work on z/TPF. We'll leave this gate open for now, with a pending decision to close
  *	it or not.
  */
-#ifdef ZOS64
-	RECORD_ERROR(state, UNSUPPORT_PLATFORM, 0);
+
+	RECORD_ERROR(state, UNSUPPORTED_PLATFORM, 0);
 	return NULL;
-#endif
 
-	/* construct the walk state and thread structures */
+	/* Maintaining original code here for potential future use.
```

Comment:
I think I would prefer to not have a large commented out piece of code.  If we decided to use this in the future we could always go back to previous revisions in the repo to get the code.  @mstoodle what are your thoughts?

Reply:
Thanks for the feedback.  I removed the commented code.